<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectralMathWidget</class>
 <widget class="QgsCollapsibleGroupBox" name="SpectralMathWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectral Math Widget</string>
  </property>
  <property name="title">
   <string>Spectral Math</string>
  </property>
  <property name="flat">
   <bool>false</bool>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <property name="collapsed">
   <bool>false</bool>
  </property>
  <property name="saveCheckedState">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>263</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QToolBar" name="mToolBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="windowTitle">
          <string>Attribute Table Toolbar</string>
         </property>
         <property name="movable">
          <bool>false</bool>
         </property>
         <property name="allowedAreas">
          <set>Qt::NoToolBarArea</set>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="floatable">
          <bool>false</bool>
         </property>
         <addaction name="actionAddFunction"/>
         <addaction name="actionRemoveFunction"/>
        </widget>
       </item>
       <item>
        <widget class="SpectralMathFunctionTableView" name="mTableView">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAddFunction">
   <property name="icon">
    <iconset resource="../../../QGIS_Images/images/images.qrc">
     <normaloff>:/images/themes/default/mActionAddExpression.svg</normaloff>:/images/themes/default/mActionAddExpression.svg</iconset>
   </property>
   <property name="text">
    <string>Add Function</string>
   </property>
   <property name="toolTip">
    <string>Adds a new Spectral Math function</string>
   </property>
  </action>
  <action name="actionRemoveFunction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../QGIS_Images/images/images.qrc">
     <normaloff>:/images/themes/default/mActionRemove.svg</normaloff>:/images/themes/default/mActionRemove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Function</string>
   </property>
   <property name="toolTip">
    <string>Removes the selected function(s)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectralMathFunctionTableView</class>
   <extends>QTableView</extends>
   <header>qps.speclib.math</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../QGIS_Images/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
